backend/scheduler/tasks.py

```diff
@@ -39,41 +35,37 @@ def create_periodic_task(
         day_of_month=day_of_month,
         month_of_year=month_of_year,
     )
-
-    logger.info(f"Creating periodic task with cron string: {cron_string}")
+    logger.info(f"Creating periodic task {task_name} with cron string: {cron_string}")
 
     # Create periodic task
-    try:
-        logger.info(f"Running periodic task addition to beat table: {schedule}")
-        PeriodicTask.objects.create(
-            crontab=schedule,
-            name=task_name,
-            task=task_path,
-            args=task_args_json,
-        )
-        logger.info(f"Completed running task addition to beat table: {task_name}")
-    except Exception as e:
-        logger.error(f"Failed to create periodic task: {e}")
+    PeriodicTask.objects.create(
```

Comment:
Shouldn't we catch a error case here ? @chandrasekharan-zipstack 
If case beat is down ?

Reply:
Its not necessary to explicitly catch and handle all such cases - it'll be bubbled up and eventually caught and logged in our global exception handler. Also if beat is down we'll still be able to create a record in the DB.
In the previous implementation we caught and logged the exception but we just let it continue silently so that behaviour is what I wanted to change

Reply:
Yeah makes sense, but the scheduler addition to db is made by beat. If beat is down the scheduler record won't be created @chandrasekharan-zipstack